src/gui/base/DropdownN.js

```diff
@@ -100,29 +116,36 @@ export class DropdownN {
 		const _contents = () => {
 			return m(".dropdown-content.plr-l.scroll.abs", {
 					oncreate: (vnode) => {
-						this.setContentHeight(vnode.dom)
 						this._domContents = vnode.dom
-						this._buttonsHeight = this._visibleChildren()
-						                          .reduce((sum, current) => sum + size.button_height, 0) + size.vpad_small * 2
-
-						const maxHeight = this._buttonsHeight + this._getFilterHeight()
-						if (this.origin) {
-							showDropdown(this.origin, this._domDropdown, maxHeight, this._width).then(() => {
-									if (this._domInput && !client.isMobileDevice()) {
-										this._domInput.focus()
-									} else {
-										const button = vnode.dom.querySelector("button")
-										button && button.focus()
+						const origin = this.origin
+						if (origin) {
+							// The dropdown-content element is added to the dom has a hidden element first.
+							// The maxHeight is available after the first onupdate call. Then this promise will resolve and we can safely show the dropdown.
+							this._maxHeightDefer.promise.then(maxHeight => {
+								showDropdown(origin, this._domDropdown, maxHeight, this._width).then(() => {
+										if (this._domInput && !client.isMobileDevice()) {
+											this._domInput.focus()
+										} else {
+											const button = vnode.dom.querySelector("button")
+											button && button.focus()
+										}
 									}
-								}
-							)
+								)
+							})
 						}
 						window.requestAnimationFrame(() => {
 							if (document.activeElement && typeof document.activeElement.blur === "function") {
 								document.activeElement.blur()
 							}
 						})
 					},
+					onupdate: (vnode) => {
+						const children = Array.from(vnode.dom.children)
+						const maxHeightOfChildren = children.reduce((accumulator, children) => accumulator + children.offsetHeight, 0) + 16
+						if (!this._maxHeightDefer.promise.isFulfilled()) {
```

Comment:
We are replacing `isFulfilled()` on another branch, we should not use it here again.